grid/recruitment/get-cv/profile.supr

```diff
@@ -0,0 +1,183 @@
+"""
+Get CV
+Gets candidate CV document.
+"""
+name = "recruitment/get-cv"
+version = "1.0.0"
+
+"""
+Get CV
+Get CV document of the candidate by candidate ID. Returns either URL or binary data of the CV document.
+"""
+usecase GetCV safe {
+  input {
+    """
+    Contact ID
+    Uniquie identifier of the candidate. Use recruitment/list-candidates profile to get candidate IDs.
+    """
+    candidateId! string!
+  }
+
+  result {
+    """
+    Curriculum Vitae
+    The CV of the candidate.
+    """
+    cv {
+      """
+      Created At
+      ISO8601 timestamp when was the CV uploaded.
+      """
+      createdAt string!
+      """
+      Document
+      Binary content and file name of the CV document.
+      """
+      document {
+        """
+        File Name
+        The file name of CV document.
+        """
+        fileName string!
+        """
+        Content
+        The content of the CV document.
+        """
+        content
+      }!
+      """
+      Document URL
+      The URL of the CV document.
+      """
+      documentUrl string!
+    }
+    """
+    Rate limit
+    Rate limit details.
+    """
+    rateLimit RateLimit
+  }
+
+  error Error
+
+  example Success {
+    input {
+      candidateId = "CANDIATE_ID"
+    }
+    result {
+      cv = {
+       createdAt = "2023-01-05T10:04:58.325Z"
+       document = {
+        fileName = "cv.pdf"
+        content = "<BinaryData>"
+       }
+      }
+    }
+  }
+
+  example Fail {
+    input {
+      candidateId = "WRONG_CANDIDATE_ID"
+    }
+    error {
+      title = "Not Found"
+      detail = "Candidate ID \"WRONG_CANDIDATE_ID\" do not exist."
+      code = "CandidateNotFound"
+    }
+  }
+}
+
+model Error {
+  """
+  Error Title
+  A short, human-readable summary of the problem type.
+  """
+  title!
+  """
+  Error Detail
+  A human-readable explanation specific to this occurrence of the problem.
+  """
+  detail
+  """
+  Code
+  Error code.
+  """
+  code! ErrorCode!
+  """
+  Rate limit
+  Rate limit details.
+  """
+  rateLimit RateLimit
+}
+
+model RateLimit {
+  """
+  Bucket
+  Different parts of API may have different rate limits.
+  Bucket identifies to which part of API the rate limits apply.
+  """
+  bucket string
+  """
+  Total requests
+  Total requests available in the time window.
+  """
+  totalRequests number
+  """
+  Remaining requests
+  Remaining requests available in the time window.
+  """
+  remainingRequests number
+  """
+  Remaining requests available in percents
+  Remaining requests available in the time window in percents.
+  """
+  remainingRequestsPercentage number
+  """
+  Reset timestamp
+  Timestamp when the rate limits will reset (in  Unix Timestamp format). Available only for providers without rolling rate limit window.
+  """
+  resetTimestamp number
+}
+
+model ErrorCode enum {
+  """
+  Bad request
+  The request was invalid or cannot be otherwise served. 
+  """
+  BadRequest
```

Comment:
I think we discussed that a HTTP Bad Request is actually a bug in the map, since a user should never be able to cause a malformed request to be sent, or am I misunderstanding the semantics of this code?

Reply:
You are right, I will get rid of this error.

Reply:
fixed